tools/lib/provision.py

```diff
@@ -121,6 +121,15 @@
     "curl",                 # Used for fetching PhantomJS as wget occasionally fails on redirects
     "moreutils",            # Used for sponge command
     "unzip",                # Needed for Slack import
+    "gconf-service",        # Puppeteer dependencies from here until xdg-utils
```

Comment:
Just move the comment to be on its own line and add a comment after `xdg-utils` noting the end of the list.